src/background.ts

```diff
@@ -154,15 +154,19 @@ async function getTotp(text: string, passphrase: string) {
   return;
 }
 
-function getDropboxToken() {
+function getBackupToken(service: string) {
+  let authUrl = '';
+  if (service === 'dropbox') {
+    authUrl =
+        'https://www.dropbox.com/oauth2/authorize?response_type=token&client_id=mmx38seexw3tvps&redirect_uri=' +
+        encodeURIComponent(chrome.identity.getRedirectURL());
+  } else if (service === 'drive') {
+    authUrl =
+        'https://accounts.google.com/o/oauth2/v2/auth?response_type=token&client_id=292457304165-ria4acohb2i875o1kmda5a31vkan7rj7.apps.googleusercontent.com&scope=https%3A//www.googleapis.com/auth/drive.file&prompt=consent&redirect_uri=' +
+        encodeURIComponent('https://authenticatortest.tk');
```

Comment:
Is this uri available? If callback uri is required, shall we use authenticator.cc instead?

Reply:
Yes, I have control over this url. Will make a blank page on authenticator.cc for redirect.